msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/TestCase1561081.java

```diff
@@ -0,0 +1,219 @@
+                .scopes(Arrays.asList(mScopes))
+                .promptParameter(Prompt.LOGIN)
+                .msalConfigResourceId(getConfigFileResourceId())
+                .build();
+
+        //AT interactive acquisition.
+        final MsalAuthResult authResult = msalSdk.acquireTokenInteractive(authTestParams_firstTry, new OnInteractionRequired() {
+            @Override
+            public void handleUserInteraction() {
+                final MicrosoftStsPromptHandlerParameters promptHandlerParameters = MicrosoftStsPromptHandlerParameters.builder()
+                        .prompt(PromptParameter.LOGIN)
+                        .loginHint(username)
+                        .sessionExpected(false)
+                        .consentPageExpected(false)
+                        .build();
+
+                new MicrosoftStsPromptHandler(promptHandlerParameters)
+                        .handlePrompt(username, password);
+            }
+        }, TokenRequestTimeout.MEDIUM);
+
+        authResult.assertSuccess();
+
+        //extract the device id claim from the access token.
+        String deviceId = (String) JWTParserFactory.INSTANCE.getJwtParser().parseJWT(authResult.getAccessToken()).get("deviceid");
+
+        //this gets the deviceId from the Ui and matches it to the deviceID obtained from the AT
+        getDeviceIdFromMyAccount(deviceId, username, password, true);
+
+        //init the release candidate BrokerHost in order to run wpj Leave
+        BrokerHost brokerHost = new BrokerHost(BrokerHost.BROKER_HOST_APK_RC);
+        if(brokerHost.isInstalled()){
+            brokerHost.uninstall();
+        }
+
+        brokerHost.install();
+        //run wpj leave
+        brokerHost.wpjLeave();
+
+        final MsalAuthTestParams authTestParams_SecondTry = MsalAuthTestParams.builder()
+                .activity(mActivity)
+                .loginHint(username)
+                .scopes(Arrays.asList(mScopes))
+                .promptParameter(Prompt.LOGIN)
+                .msalConfigResourceId(getConfigFileResourceId())
+                .build();
+
+
+        //acquire the token interactively for a second time.
+         final MsalAuthResult authResult2 = msalSdk.acquireTokenInteractive(authTestParams_SecondTry, new OnInteractionRequired() {
+            @Override
+            public void handleUserInteraction() {
+                final MicrosoftStsPromptHandlerParameters promptHandlerParameters = MicrosoftStsPromptHandlerParameters.builder()
+                        .prompt(PromptParameter.LOGIN)
+                        .loginHint(username)
+                        .sessionExpected(false)
+                        .consentPageExpected(false)
+                        .build();
+
+                new MicrosoftStsPromptHandler(promptHandlerParameters)
+                        .handlePrompt(username, password);
+            }
+        }, TokenRequestTimeout.MEDIUM);
+
+        authResult2.assertSuccess();
```

Comment:
I think per the test, we want to again use the JWT Parser here and confirm that now the token does NOT contain `deviceid`